src/shim/shim-config.ts

```diff
@@ -203,8 +204,11 @@ class Config implements IConfig {
             options.logInstrumentationOptions = {
```

Comment:
Is a good time to remove logInstrumentationOptions  entirely and only use instrumentationOptions, this what will be supported in distro in the future

Reply:
Removed the other `logInstrumentationOptions` and cleaned up the tests.